less/confidenceSlider.less

```diff
@@ -1,6 +1,11 @@
 .confidenceslider {
-  &__fill-linked {
-    height: 100%;
-    background-color: yellow;
+  .slider {
+    &__item-input-fill-linked {
+      position: absolute;
+      left: 0;
+      height: @slider-track-height;
```

Comment:
`@slider-track-height` is undefined

Reply:
Hi @kirsty-hames yes I do see that it is, however would this mean that unless you have both components within a course that confidence slider will not receive the correct styling?

The inheritance that lies within view/model is fine but I don't think this is the same for .less files. 

I may be wrong or missing something here so please let me know!